sogs/__main__.py

```diff
@@ -51,6 +57,27 @@
     metavar='SESSIONID',
     help="Delete the the given Session ID(s) as moderator and admins of the room given by --rooms",
 )
+actions.add_argument('--set-perms', help="Sets permissions for the room given by --rooms")
+ap.add_argument(
+    '--users',
+    help="Users to set permissions for with --set-perms; if omitted, set global perms",
+    nargs='+',
+    metavar='SESSIONID',
+)
+ap.add_argument(
+    "--add-perms",
```

Comment:
Thoughts on calling this "--grant-perms" instead of "--add-perms"?  (And likewise revoke instead of remove)?

Reply:
utter indifference